src/sections/Contributors/Contributors.jsx

```diff
@@ -3,8 +3,16 @@ import "./Contributors.css";
 import DiscordIcon from "../../assets/icons/discord-fill.svg";
 import GithubIcon from "../../assets/icons/github-fill.svg";
 import LinkedInIcon from "../../assets/icons/linkedin-box-fill.svg";
+import { Grid, Avatar, Typography, Stack } from "@mui/material";
+import Data from "../../data";
+import Contributor from "../../components/Contributor";
+
+const Contributors = () => {
+	// Map through contributors from data.js
+	const contributors = Data.map((contributor) => {
+		return <Contributor key={contributor.id} {...contributor} />;
+	});
 
-const Contributors = (props) => {
 	return (
 		<div className="con-body">
```

Comment:
Would like for this to be more semantically correct. Instead of using just divs

Reply:
Corrected it to be body with 1 section, removed the unnecessary divs